src/include/miopen/find_solution.hpp

```diff
@@ -509,4 +509,4 @@ struct SolverContainer
 } // namespace solver
 } // namespace miopen
 
-#endif
+#endif
```

Comment:
Also unintended modification. Please check whatever comment points out, is also happening in other codes.

Reply:
I restored.